peer_test.go

```diff
@@ -355,6 +356,8 @@ func (pt *peerSelectionTest) setupAffinity(t testing.TB) {
 		}(affinity)
 	}
 
+	// use ping to build connection without sending call req.
+	pt.makePing(pt.servers[pt.numAffinity-pt.numAffinityWithNoCall], hostport)
```

Comment:
let's rename `makePing` to `sendPing` (optionally rename `makeCall` to `sendCall`)

You're only making a ping request from a single node right now.